### PR TITLE
Record in database when end time is "--"

### PR DESCRIPTION
**Pull request by ekimekim**
We need this so that reverse sync reproduces these values correctly.

To handle this in the database, we have a composite type (dashed: boolean, value: timestamp).
Value is always valid and is equivalent to the old timestamp column,
but must be equal to start_time if dashed is true.

The only place we directly reference this column outside sheetsync is thrimshim, where we
always consider the value only.

### CODE REVIEW

File: thrimshim/thrimshim/main.py [hunk 1]
```diff
@@ -408,6 +413,7 @@ def normalize(value):
 		return 'Sheet columns have changed since editing has begun. Please review changes\n' + changes, 409
 
 	if new_row['state'] == 'MODIFIED':
+		missing = []
```
Comment (ekimekim) [inline thrimshim/thrimshim/main.py:416]: Drive-by fix here for a bug I randomly spotted.
Previously we didn't define `missing` before using it here, but there happened to be a `missing` defined previously which would always be empty, so as long as no columns were ever missing it would still work.

**Comment by chrusher** [comment]
Looks ok to me
